### DISCUSSION TITLE
Slide Show only showing each picture only once

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#268] [Q&A]
It looks like that the slide show stopp if all pictures schown. Can you do it in a loop ?

**Comment by mgineer85** [comment]
I think I found it stopping at one point also. But as I usually do not use the feature I did not care too much.
The original implementation is by @DominicWindisch , maybe he noticed also and could take a look?

**Comment by DominicWindisch** [comment]
I haven't had time to work on anything photo booth related in the past time. 🙁

The slideshow (if I remember correctly) creates a randomized order of all image indices up to the last one and then starts showing them in order. Shouldn't be too hard to create a new such order after showing all images once. It implemented in the frontend though:  [here](https://github.com/photobooth-app/photobooth-frontend/blob/959bfe98d80badc29f97c270036369abc3b8cf6e/src/components/GalleryImageDetail.vue#L367).

I don't have the time at the moment to fix and test it unfortunately.
